ckanext/unaids/blueprints/ape_data_receiver.py

```diff
@@ -6,6 +6,7 @@
 
 ape_data_receiver = Blueprint("ape_data_receiver", __name__)
 
+
```

Comment:
It's usually best to avoid including lines like this in your PR.  Keep your PR nice and clean with only the necessary changes for your specific feature. If there are major PEP problems, then they should be addressed in their own PR. As it is, I'm not too fussed about this one line, but just bear this in mind for the future.  Always review your own PR and clear out any unnecessary changes that have leaked their way into the PR.